modules/HardwareDrivers/PowerSupplies/Huawei_V100R023C10/fusion_charger_lib/huawei-fusioncharge-driver/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.22)
```

```suggestion
cmake_minimum_required(VERSION 3.16)
```

https://github.com/EVerest/everest-core/pull/1336#discussion_r2309691980 still applies